templates/tips/categories.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="../../static/css/style.css">
+  </head>
+  <body>
+    <script src="index.js"></script>
```

Comment:
Where is index.js located?

Reply:
@orIsrael Please respond to this conversation

Reply:
I've deleted it thank you:)